datavault-webapp/src/main/webapp/WEB-INF/freemarker/vaults/index.ftl

```diff
@@ -238,8 +238,7 @@
             $.validator.addMethod(
                 "date",
                 function(value, element) {
-                    // put your own logic here, this is just a (crappy) example
-                    return value.match(/^\d{4}-\d{2}-\d{2}$/);
+                    return this.optional(element) || value.match(/^\d{4}-\d{2}-\d{2}$/);
```

Comment:
I've already push something for fixing this and it seems to work so I suggest you remove changes to this file and just push the rest?